app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,5 @@
+<footer class="page-footer font-small blue">
+  <div class="footer-copyright text-center py-3">© 2019 Copyright:
```

Comment:
There is no change which we did for this year. Did you push all changes to remote repository?